testdata/project-v2-addon/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM golang:1.13 as builder
+FROM golang:1.16 as builder
```

Comment:
This doesn't seem right. I don't think we should be changing the version for v2 because that was built using 1.13.